docroot/themes/contrib/civictheme/theme-settings.php

```diff
@@ -634,3 +635,388 @@ function _civictheme_path_field_description($original_path, $fallback_path) {
+  $form['colors'] = [
+    '#type' => 'details',
+    '#title' => t('Colors'),
+    '#weight' => 40,
+    '#open' => TRUE,
+    '#tree' => TRUE,
+  ];
+
+  $form['colors']['use_brand_colors'] = [
+    '#title' => t('Use Brand colors'),
+    '#type' => 'checkbox',
+    '#default_value' => theme_get_setting("colors.use_brand_colors") ?? TRUE,
+  ];
+  // Brand.
+  $form['colors']['brand'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Brand colors'),
+    '#tree' => TRUE,
+    '#states' => [
+      'visible' => [
+        ':input[name="colors[use_brand_colors"]' => ['checked' => TRUE],
+      ],
+    ],
+  ];
+
+  // Brand light.
+  $form['colors']['brand']['light'] = [
+    '#type' => 'fieldset',
+    '#tree' => TRUE,
+    '#attributes' => [
+      'class' => 'color-light',
+    ],
+  ];
+  $form['colors']['brand']['light']['brand1'] = [
+    '#type' => 'color',
+    '#title' => t('Brand1'),
+    '#default_value' => theme_get_setting("colors.brand.light.brand1") ?? '#00698f',
+  ];
+  $form['colors']['brand']['light']['brand2'] = [
+    '#type' => 'color',
+    '#title' => t('Brand2'),
+    '#default_value' => theme_get_setting("colors.brand.light.brand2") ?? '#e6e9eb',
+  ];
+  $form['colors']['brand']['light']['brand3'] = [
+    '#type' => 'color',
+    '#title' => t('Brand3'),
+    '#default_value' => theme_get_setting("colors.brand.light.brand3") ?? '#121313',
+  ];
+
+  // Brand dark.
+  $form['colors']['brand']['dark'] = [
+    '#type' => 'fieldset',
+    '#tree' => TRUE,
+    '#attributes' => [
+      'class' => 'color-dark',
+    ],
+  ];
+  $form['colors']['brand']['dark']['brand1'] = [
+    '#type' => 'color',
+    '#title' => t('Brand1'),
+    '#default_value' => theme_get_setting("colors.brand.dark.brand1") ?? '#61daff',
+  ];
+  $form['colors']['brand']['dark']['brand2'] = [
+    '#type' => 'color',
+    '#title' => t('Brand2'),
+    '#default_value' => theme_get_setting("colors.brand.dark.brand2") ?? '#003a4f',
+  ];
+  $form['colors']['brand']['dark']['brand3'] = [
+    '#type' => 'color',
+    '#title' => t('Brand3'),
+    '#default_value' => theme_get_setting("colors.brand.dark.brand3") ?? '#00698f',
+  ];
+
+  // Palette.
+  $form['colors']['palette'] = [
+    '#type' => 'details',
+    '#title' => t('Palette colors'),
+    '#open' => TRUE,
+    '#tree' => TRUE,
+  ];
+
+  // Palette light.
+  $form['colors']['palette']['light'] = [
+    '#type' => 'fieldset',
+    '#tree' => TRUE,
+    '#attributes' => [
+      'class' => 'color-light',
+    ],
+  ];
+  // Palette light typography.
+  $form['colors']['palette']['light']['typography'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Typography'),
+    '#tree' => TRUE,
+  ];
+  $form['colors']['palette']['light']['typography']['heading'] = [
+    '#type' => 'color',
+    '#title' => t('Heading'),
+    '#default_value' => theme_get_setting("colors.palette.light.heading") ?? '#000000',
```

Comment:
please replace default colors with `CIVICTHEME_PALETTE_COLOR_LIGHT_DEFAULT` with value `#000000` constant (add the constant to the `constants.inc` file) for brand colors

Reply:
Updated